components/style/properties/properties.mako.rs

```diff
@@ -1145,7 +1145,10 @@ impl ComputedValues {
     #[inline]
     pub fn is_multicol(&self) -> bool {
         let style = self.get_column();
-        style.column_count.0.is_some() || style.column_width.0.is_some()
```

Comment:
It checks for column_count under the `Either::Second(_auto)` match arm.  If the first match arm is followed, there's no need to check column count.